code/modules/mob/mob_defines.dm

```diff
@@ -70,9 +70,6 @@
 	*/
 	var/obj/screen/zone_selector/zone_sel = null
 
-	/// Cursor icon used when holding shift over things.
-	var/examine_cursor_icon = 'icons/effects/mouse_pointers/examine_pointer.dmi'
```

Comment:
for parity with the prior system we could specify an examine cursor *decl* on mob instead

Reply:
mostly that it's a mob var that can be overridden